package dersCalisma2;

import java.io.IOException;

public class trycatchdene {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
		try {
		
		int a=3-3;
		int b=2/a;
		System.out.println(""+b);

	}
		catch(ArithmeticException e)
		{
			e.printStackTrace();
		}
finally {}
		
}}
